whippet.php

```diff
@@ -378,6 +378,41 @@ function signal_handler($signal) {
 
 ////Whippet START
 
+// If whippet-server has wrapped the WordPress instance, WPS_XY has been defined already.
+if (defined('WPS_LOCATION')) {
```

Comment:
if this is intended to represent whippet-server having wrapped the code then this would be better represented by extracting a function with a meaningful name. This would do away with the need for the above comment.